worker/include/RTC/KeyFrameRequestManager.hpp

```diff
@@ -0,0 +1,98 @@
+#ifndef MS_KEY_FRAME_REQUEST_MANAGER_HPP
+#define MS_KEY_FRAME_REQUEST_MANAGER_HPP
+
+#include "handles/Timer.hpp"
+#include <map>
+
+static uint16_t KeyFrameWaitTime{ 2000 };
```

Comment:
This is creating a global constant visible to any source file including this header. I think we should define the class constructor in the CPP file and have declare there required constants.